create-keystone-app/starter/README.md

```diff
@@ -0,0 +1,50 @@
+# Keystone Project Parter
+
+Welcome to Keystone!
+
+Run
+
+```
+npm keystone dev
+```
+
+To view the config for your new app, look at [./keystone.ts](./keystone.ts)
+
+This project starter is designed to give you a sense of the power keystone can offer you, and show off some of its main features. It's also a pretty simple setup if you want to build out from it.
+
+We recommend you use this alongside our [getting started walkthrough](https://keystonejs.com/docs/walkthroughs/getting-started-with-create-keystone-app) which will walk you through what you get as part of this starter.
+
+If you want an overview of all the features keystone offers, check out our [why keystone](https://keystonejs.com/why-keystone) page.
+
+## Some Quick Notes On Getting Started
+
+### Changing the database
+
+We've set you up with an [sqlite database](https://keystonejs.com/docs/apis/config#sqlite) for ease-of-use. If you're wanting to use postgres, you can!
```

Comment:
Nitpicking, but there are a few references that we usually refer to in the right casing, example being `sqlite` usually stated as `SQLite` when not in configuration or code examples.

Reply:
```suggestion
We've set you up with an [SQLlite database](https://keystonejs.com/docs/apis/config#sqlite) for ease-of-use.
```